install/jstl/docs/ReleaseNotes-jstl.html

```diff
@@ -1,190 +1,190 @@
-          <p> </p>
-          <hr> <b>Note</b> - Jakarta EE Platform compatibility is not required;
-          this TCK only requires a container that supports Jakarta EE Standard
-          Tag Library 2.0 and its requirements.
-          <hr>
-          <p></p>
-        </li>
-        <li>Jakarta EE Standard Tag Library TCK tests run on the following
-          platforms:
-          <ul>
-            <li>Alpine Linux v3.12</li>
-            <li>CentOS Linux 7</li>
-          </ul>
-        </li>
-        <hr> </a>
-      <h2><a name="ReferenceRuntime"></a><a name="install_setup_run">Installing,
-          Setting Up, and Running the A Standard Tag Library for JavaServer
-          Pages<small><sup>TM</sup></small> TCK</a></h2>
-      <p>Refer to the<em> Jakarta EE Standard Tag Library TCK 2.0 User's Guide</em>
-        (<a href="./html-usersguide/title.html" title="Jakarta EE Standard Tag Library TCK 2,0 User's Guide (HTML)">HTML</a>,
-        <a href="./pdf-usersguide/Jakarta-Standard-Tag-Library-TCK-Users-Guide.pdf"

+        Implementation for Jakarta EE Standard Tag Library 3.0</a> for
+      additional information.</p>
+    <hr> <a name="ReferenceRuntime">
+      <h2>Compatible Implementation for A Standard Tag Library 3.0</h2>
+      <p>The designated Runtime for compatibility testing of implementations
+        based upon the Jakarta EE Standard Tag Library Specification 3.0 is as
+        follows:</p>
+    </a>
+    <ul>
+      <a name="ReferenceRuntime">
+        <li>This Jakarta EE Standard Tag Library TCK has been built on JDK 11
+          and has been tested with Java Standard Edition 11 and 17.</li>
+        <li>Eclipse Compatible Implementation (CI) of Jakarta EE Standard Tag
+          Library 3.0 available as a standalone implementation from Jakarta EE
+          Specifications web site, or included as part of the Jakarta EE 10.0
+          compatible implementation.
+          <p> </p>
+          <hr> <b>Note</b> - Jakarta EE Platform compatibility is not required;
+          this TCK only requires a container that supports Jakarta EE Standard
+          Tag Library 3.0 and its requirements.
+          <hr>
+          <p></p>
+        </li>
+        <li>Jakarta EE Standard Tag Library TCK tests run on the following
+          platforms:
+          <ul>
+            <li>TBD and possible remove the following two entries</li>
+            <li>Alpine Linux v3.12</li>
+            <li>CentOS Linux 7</li>
+          </ul>
+        </li>
+        <hr> </a>
+      <h2><a name="ReferenceRuntime"></a><a name="install_setup_run">Installing,
+          Setting Up, and Running the A Standard Tag Library for JavaServer
+          Pages<small><sup>TM</sup></small> TCK</a></h2>
+      <p>Refer to the<em> Jakarta EE Standard Tag Library TCK 3.0 User's Guide</em>
+        (<a href="./html-usersguide/title.html" title="Jakarta EE Standard Tag Library TCK 2,0 User's Guide (HTML)">HTML</a>,
```

Comment:
do we need to retain 2,0 or change to 3.0 or 3,0?

Reply:
I think it should be `3.0`